### ISSUE TITLE
"Cancel" button doesn't work as expected on Meeting Request page

### ISSUE THREAD

**Issue by smyslov** [ietf-tools/datatracker#4007]
### Describe the issue

When you press "Cancel" button at the end of a meeting request page for a particular WG, the result is that you are being scrolled to the top of the page. Nothing more happened. Currently the only way to escape from the meeting request page is to request a session (or press browser's "Back" button).

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by Spectre17** [comment]
I logged into the datatracker as Valery. I went to Meetings->Request a Session (/secr/sreq). I selected "dots" as the only available option to Valery at this time (for IETF 114). I filled in the form on /secr/sreq/dots/new to the extent necessary (but really, it shouldn't matter since I'm not going to request the session). I clicked on the "Cancel" button. It took me back to  /secr/sreq. I did not observe it simply jumping back to the top of the page. I believe the behavior observed is what's desired. Has something changed in this component in the interim, or am I simply misunderstanding Valery's issue?
